### ISSUE TITLE
More options for sorting contributors

### ISSUE THREAD

**Issue by tenshiAMD** [all-contributors/cli#320]
I suggest having more options for sorting contributors.

Options:
- none (default)
- username 
- number of contributions

Types:
- ascending - without minus sign  (default)
- descending - with minus sign

### Proposed format
```
contributorsSortBy: "-username"
```

### Old format
```
"contributorsSortAlphabetically": false
```

**Comment by tenshiAMD** [comment]
@gr2m @Berkmann18 what do you think? Please let me know. Thanks! 🎉

**Comment by Berkmann18** [comment]
I agree, having the field name or something meaningful like `"username"`; I like the idea behind the `-` for descending although that would definitely need docs as it's not intuitive.
